src/components/homepage/HeroSection.jsx

```diff
@@ -33,197 +31,169 @@ const EditorHeroSection = ({
-              <input
-                type="radio"
-                id="radio-dropdown"
-                name="hero-type"
-                defaultValue="dropdown"
-                onChange={handleHighlightDropdownToggle}
-                checked={!!dropdown}
-              />
-              Dropdown
-            </label>
-          </div>
-          <span className={elementStyles.info}>
-            Note: you can only have either Key Highlights+Hero button or a Hero
-            Dropdown
-          </span>
-        </div>
-        <div>
-          {dropdown ? (
-            <>
-              <HeroDropdown
-                title={dropdown.title}
-                options={dropdown.options}
-                sectionIndex={sectionIndex}
-                createHandler={createHandler}
-                deleteHandler={(event) =>
-                  deleteHandler(event, "Dropdown Element")
-                }
-                onFieldChange={onFieldChange}
-                displayDropdownElems={displayDropdownElems}
-                displayHandler={displayHandler}
-                errors={errors}
-              />
-            </>
-          ) : (
-            <>
-              <HeroButton
-                button={button}
-                url={url}
-                sectionIndex={sectionIndex}
-                onFieldChange={onFieldChange}
-                errors={errors.sections[0].hero}
-              />
-              <Droppable droppableId="highlight" type="highlight">
-                {(droppableProvided) => (
-                  <div
-                    className={styles.card}
-                    ref={droppableProvided.innerRef}
-                    {...droppableProvided.droppableProps}
-                  >
-                    {highlights && highlights.length > 0 ? (
-                      <>
-                        <b>Hero highlights</b>
-                        {highlights.map((highlight, highlightIndex) => (
-                          <Draggable
-                            draggableId={`highlight-${highlightIndex}-draggable`}
-                            index={highlightIndex}
+        </>
+      ) : (
+        <>
+          <HeroButton
+            button={button}
+            url={url}
+            sectionIndex={sectionIndex}
+            onFieldChange={onFieldChange}
+            errors={errors.sections[0].hero}
+          />
+          <Droppable droppableId="highlight" type="highlight">
+            {(droppableProvided) => (
+              <div
+                className={styles.card}
+                ref={droppableProvided.innerRef}
+                {...droppableProvided.droppableProps}
+              >
+                {highlights && highlights.length > 0 ? (
+                  <>
+                    <b>Hero highlights</b>
+                    {highlights.map((highlight, highlightIndex) => (
+                      <Draggable
+                        draggableId={`highlight-${highlightIndex}-draggable`}
```

Comment:
we are using an array index as an id here -> could we use a uuid/ anything else unique instead? Worried that this will introduce bugs considering we are reordering the elements, which is specifically where using array as index key _will_ break.

Reply:
hmm sounds good! this has been using this `draggableId` all this time (even on `production`) so idt it'll break but this is a good change to make regardless

Reply:
fyi, gonna make this change in `Editable.DraggableAccordionItem` - this particular `draggableId` will be left as is becaues the abstraction we want to present is all draggable stuff goes through the new `Editable`. This particular `HeroSection.jsx` will be entirely removed later on

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/1404/commits/91ba5cf30a8ca374903d7a6f8ac640773b006955